srcpkgs/mpv-mpris/template

```diff
@@ -16,6 +16,6 @@ checksum=673aff031e7cc741edea68d7b4b0103d7b031d4a55755abb9e1be5dd4ec4e969
 
 do_install() {
 	vlicense LICENSE
-	vmkdir usr/lib/${pkgname}
-	vinstall mpris.so 0644 usr/lib/${pkgname}
+	vmkdir etc/mpv/scripts
+	vinstall mpris.so 0644 etc/mpv/scripts/
```

Comment:
Does this enable it by default? In my setup I have `~/.config/mpv/scripts/mpris.so -> /usr/lib/mpv-mpris/mpris.so`, and it just works. Having it enabled for everyone wouldn't be ideal for a multiuser setup or similar.

Reply:
Also, would this result in errors for people who aren't using a D-Bus bus?

Reply:
What does that do exactly (I added vdoc anyways)? I was thinking something more along the lines of using `INSTALL.msg`

Reply:
It adds the README to the package, so people can hunt for how to enable it in `/usr/share/doc/mpv-mpris`.